plugins/inventory/ovirt.py

```diff
@@ -126,7 +126,7 @@ def _get_dict_of_struct(self, vm):
             'os_type': vm.os.type,
             'template': self.connection.follow_link(vm.template).name,
             'creation_time': str(vm.creation_time),
-            'creation_time_timestamp': vm.creation_time.timestamp(),
+            'creation_time_timestamp': vm.creation_time.strftime("%s.%f"),
```

Comment:
This changes the type from `float` to `str` - I don't have the bigger picture, so not sure if that's ok. Nit - I can also see more single quoted strings throughout the module (although the quotes are not 100% consistent, so up to you).

Reply:
I suppose you could just wrap it in `float(vm.creation_time.strftime("%s.%f"))`  that way python3 does not change behavior and python2 will have same behavior

Reply:
> I suppose you could just wrap it in `float`...

Yes, this will give you the same type for both py2 and py3, but looks really wrong - we're going from numeric type, converting it to string and then back to numeric one. Floating point numbers' representation in memory is power-of-2-based and not power-of-10-based, so parsing from strings can lead to unpredictable results.

AFAICS, this code was introduced [here](https://github.com/oVirt/ovirt-ansible-collection/commit/f86f49730f89d7606f417bd440da85d74c7a2c90), meaning that this is not a regression on py2 - it simply never worked.

Some important questions that would need an answer:
- how is that timestamp used? can we introduce breakage of type and/or precision here?
- do we absolutely, positively need microsecond precision?

One solution I see as reasonable is to use `time.mktime(vm.creation_time.timetuple())`, but the `time_struct` that is created in between doesn't have the microseconds part, so we end up with zero-rounded floats, i.e. `1598263100.0`.